### ISSUE TITLE
hybrid loss implementation in UNET3+ example

### ISSUE THREAD

**Issue by atgc1984** [yingkaisha/keras-unet-collection#15]
Thank you for updating the demo for hybrid loss implementation. 

Are the losses offered in this package designed for multi-class segmentation? I guess they should be different from those binary cases (like overall iou = averaged sum of class iou). 
Also the comments for _iou_seg_ indicate that the input should be box coordinates, which is confusing from the demo since the model output is tensor of probabilities with length of n_labels.

**Comment by yingkaisha** [comment]
I've fixed the documentation of `iou_seg` (originally duplicated from the `iou_box`). Its usage is as in the Keras cross-entropy loss.
